### PR TITLE
[BRN-183] feat: update to multichain add ftm protocols & strategies

### PR DESCRIPTION
**Pull request by DarkGhost7**

### CODE REVIEW

File: data/strategies/250/VoterProxy.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "$schema": "strategy",
+    "name": "Curve Boost",
+    "description": "Supplies {{token}} to [Curve Finance](https://ftm.curve.fi) and stakes it in gauge to collect any available tokens and earn enhanced CRV rewards thanks to [locked CRV boost](https://resources.curve.fi/guides/boosting-your-crv-rewards). Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy.",
+    "addresses": [
+      "0x65fCDb59D9dc496Fd957525B74ed427fAC110E85",
+      "0x98B058b2CBacF5E99bC7012DF757ea7CFEbd35BC",
+      "0x309Bac9cC01912decf6EC286c1F17Db99926c44F"
+    ],
+    "protocols": ["CurveFinance"]
+  }
```
Comment (jstashh) [inline data/strategies/250/VoterProxy.json:11]: spacing looks off and extra final line

File: data/strategies/250/TarotReinvest.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "name": "Tarot Reinvest",
+    "description": "Supplies {{token}} to [Tarot](tarot.to) to earn TAROT. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy.",
+    "addresses": [
+      "0x1922Fde0C9f09cD1AEAe20C6021a2c18a9CBD589",
+      "0x47C004db0Dd0dCe726827CAAac71b660FFF14fAb",
+      "0xFb96225463d2A53DC1C8747Ddb2e019BF4391897",
+      "0xd8CA58CaD88Ac5Dc3b47B7BFfC7ce899F2c4acD8"
+    ],
+    "protocols": ["Tarot"]
+  }
```
Comment (jstashh) [inline data/strategies/250/TarotReinvest.json:12]: spacing looks off

File: data/strategies/250/SpookyswapReinvest.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "strategy",
+    "name": "Spookyswap Reinvest",
+    "description": "Supplies {{token}} to [Spookyswap](spookyswap.finance) to earn BOO. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy.",
+    "addresses": [
+      "0xf8c08cE855D1ABA492202ecf47eaa3d2a7DE2eC5"
+    ],
+    "protocols": ["Spookyswap"]
+  }
```
Comment (jstashh) [inline data/strategies/250/SpookyswapReinvest.json:9]: spacing looks off

File: data/strategies/250/SpookyJointLiquidityProvider.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    "$schema": "strategy",
+    "name": "Spooky Joint Liquidity Provider",
+    "description": "Supplies {{token}} as half of a join liquidity provider for a pool on [Spookyswap](spookyswap.finance). Rewards are harvested, split between the two pairs, sold for more {{token}}, and re-deposited into the strategy.",
+    "addresses": [
+      "0x9963F18cA33e6CC0Bb8032D2B9E2511d4931037e",
+      "0xecc1DFF0C8450A5A16E56211c43182B95580f9Cf",
+      "0xaA944D01b361b25a11F185C61530937a7a9C47a6"
+    ],
+    "protocols": ["Spookyswap"]
+  }
```
Comment (jstashh) [inline data/strategies/250/SpookyJointLiquidityProvider.json:11]: spacing looks off

File: data/strategies/250/ScreamLenderOptimizer.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    "$schema": "strategy",
+    "name": "Scream Lender Optimizer",
+    "description": "Supplies {{token}} to [Scream](scream.sh) to generate interest and earn SCREAM tokens. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy.",
+    "addresses": [
+      "0x5920892f23967728E16135c21aB8BeEC8C548927",
+      "0x754133e0f67CB51263d6d5F41f2dF1a58a9D36b7"
+    ],
+    "protocols": ["Scream"]
+  }
+
```
Comment (jstashh) [inline data/strategies/250/ScreamLenderOptimizer.json:11]: spacing looks off and extra final line

File: data/strategies/250/EsterReinvest.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "strategy",
+    "name": "Ester Reinvest",
+    "description": "Supplies {{token}} to [Ester.Finance](https://ester.finance) to earn BOO. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy.",
+    "addresses": [
+      "0x0Edd8F5786f2AFeBa9fa56a65487bA3F6264AAA9"
+    ],
+    "protocols": ["EsterFinance"]
+  }
```
Comment (jstashh) [inline data/strategies/250/EsterReinvest.json:9]: spacing looks off

**Review by jstashh — APPROVED** [review]

**Comment by jstashh** [comment]
@DarkGhost7 we ok to merge this? We will need to adapt the sdk to use it at some point

**Comment by DarkGhost7** [comment]
Yes, it's ready to merge

**Pull request merged**